Bulk RNA-seq cheatsheet

cbethell opened the pull request:
As stated on closed PR #137:
"This is a draft of the proposed ideas for the module cheatsheets.

I added links to each piece of code (some of which I have yet to add)
I also added a diagram to explain the command line commands a little differently (I still have a diagram or 2 to add)"

jaclyn-taroni commented on module_cheatsheets/Bulk_RNA-seq_Cheatsheet.md:
```diff
@@ -0,0 +1,63 @@
+
+
+### Useful command line commands:
+<br>
```
Can you either use examples within the training structure + Docker image or note that this is a departure from that structure you are using for illustrative purposes?

cbethell replied:
@jaclyn-taroni Including headers is a great idea, I'll do that along with the links to their documentation...as well as ensure that all of the links correspond to the package versions used in the training module. 

> Can you either use examples within the training structure + Docker image or note that this is a departure from that structure you are using for illustrative purposes?

Do you have a preference of which route I go here? The examples I used here were meant to be the simplest possible, however, of course there can also be some usefulness in relating them back to training structure.

cbethell commented:
@jaclyn-taroni Including headers is a great idea, I'll do that along with the links to their documentation...as well as ensure that all of the links correspond to the package versions used in the training module. 

> Can you either use examples within the training structure + Docker image or note that this is a departure from that structure you are using for illustrative purposes?

Do you have a preference of which route I go here? The examples I used here were meant to be the simplest possible, however, of course there can also be some usefulness in relating them back to training structure.

jaclyn-taroni commented:
If I think about what I would want in a simplest possible example, the file structure would look something like
```
01-data_cleaning.Rmd
01-data_cleaning.nb.html
02-analysis.Rmd
02-analysis.nb.html
data
  ├── file_1.txt
  └── file_2.txt
```
Where either you illustrate navigating into and back up out of `data` or this all lives in a directory called something like `my-project` that you navigate into in your example.

cbethell commented:
> If I think about what I would want in a simplest possible example, the file structure would look something like
> 
> ```
> 01-data_cleaning.Rmd
> 01-data_cleaning.nb.html
> 02-analysis.Rmd
> 02-analysis.nb.html
> data
>   ├── file_1.txt
>   └── file_2.txt
> ```
> 
> Where either you illustrate navigating into and back up out of `data` or this all lives in a directory called something like `my-project` that you navigate into in your example.

Okay, I'll look into implementing both of those ideas and see which one may be better suited for this purpose

jaclyn-taroni commented:
You can also use the training structure if you think it's faster than throwing together these dummy files.

cbethell commented:
In the most recent commit [837a4d1](https://github.com/AlexsLemonade/training-modules/pull/138/commits/837a4d1d36ee49839366f617b525566d5bcebbd4):
-Added headers before each table
-Changed the links to correspond with the versions used in the training modules where possible (the `DESeq2` functions for the version used were not found) 
-Changed the command line example to incorporate the training-structure, but did not create a folder in this repo to correspond with the example as I notice the lack of a `data` folder in this repo. I added a note encouraging participants to follow along with the example...would this be suffice or should I still take a step further?

jaclyn-taroni commented on module_cheatsheets/Bulk_RNA-seq_Cheatsheet.md:
```diff
@@ -0,0 +1,87 @@
+| [`stringr`](https://www.rdocumentation.org/packages/stringr/versions/1.3.1)            |[`word()`](https://www.rdocumentation.org/packages/stringr/versions/1.3.1/topics/word)       | Word                            | Extracts words from a character vector                         |                                                                                        
+| [`readr`](https://www.rdocumentation.org/packages/readr/versions/1.3.0)                 |[`write_rds()`](https://www.rdocumentation.org/packages/readr/versions/1.3.0/topics/read_rds) | Write RDS                      | Writes data to a .rds output file                                 |                                                                                         
+| [`tximport`](https://bioconductor.org/packages/release/bioc/vignettes/tximport/inst/doc/tximport.html)              | [`tximport()`](https://www.rdocumentation.org/packages/tximport/versions/1.11.6/topics/tximport)           | Tximport           | Imports transcript-level estimates and summarizes abundances, counts, and transcript lengths to the gene-level by default   |
+| [`dplyr`](https://www.rdocumentation.org/packages/dplyr/versions/0.7.8)                 | [`pull()`](https://www.rdocumentation.org/packages/dplyr/versions/0.7.8/topics/pull)                      | Pull                              | Extracts a variable as a vector                                          |                                                                                       
```
```suggestion
| [`dplyr`](https://www.rdocumentation.org/packages/dplyr/versions/0.7.8)                 | [`pull()`](https://www.rdocumentation.org/packages/dplyr/versions/0.7.8/topics/pull)                      | Pull                              | Extracts a variable (column) as a vector                                          |                                                                                       
```

jaclyn-taroni commented on module_cheatsheets/Bulk_RNA-seq_Cheatsheet.md:
```diff
@@ -0,0 +1,87 @@
+---
```
If moving to MacDown - I believe you'll want to take this `output` section out.

cbethell commented:
@jaclyn-taroni Great! I'll make those changes now.
